v2/azure-arm.yaml

```diff
@@ -1274,6 +1277,35 @@ objectModelConfiguration:
       VirtualNetworks_VirtualNetworkPeering:
         $exportAs: VirtualNetworksVirtualNetworkPeering
         $supportedFrom: v2.0.0-alpha.1
+    2022-07-01:
+      PrivateEndpoint:
+        $export: true
+        $supportedFrom: v2.0.0
+      PrivateEndpointConnection_STATUS:
+        $isResource: true
```

Comment:
Curious about this - making a note to myself to understand this workflow

Reply:
Do you have anywhere in tests where you show how this is used? How does this work? it seems weird for this to be a resource, and it also feels weird for this to be its name.

I assume this has something to do with showing the status of the PrivateEndpointConnection (This is a GET-only resource in ARM, basically?)

Can you walk me through how the user uses this? I don't actually see an example of it in the samples?